mods/ra/maps/allies-03a/map.yaml

```diff
@@ -1311,3 +1311,5 @@ Actors:
 Rules: ra|rules/campaign-rules.yaml, ra|rules/campaign-tooltips.yaml, ra|rules/campaign-palettes.yaml, rules.yaml
 
 Weapons: weapons.yaml
+
+Translations: ra|languages/lua/en.ftl
```

Comment:
`languages/campaign/*` to match the `campain-*.yaml` rule overrides?

Reply:
I don't want to make a separate folder for mini games.

Reply:
I would expect that minigames would define their translations directly in their map.yaml?

Reply:
Then why do you want a separate folder anyway? The folder structure has already been determined by the #20265 pull request. I say we stay consistent for now. We might have to change this into something entirely different anyway when we hook up the web translation automatism in the future.